ras-record.c

```diff
@@ -212,9 +212,11 @@ static const struct db_fields arm_event_fields[] = {
 		{ .name="mpidr",		.type="INTEGER" },
 		{ .name="running_state",	.type="INTEGER" },
 		{ .name="psci_state",		.type="INTEGER" },
+#ifdef HAVE_AMP_NS_DECODE
 		{ .name="err_info",		.type="BLOB"	},
 		{ .name="context_info",		.type="BLOB"	},
 		{ .name="vendor_info",		.type="BLOB"	},
+#endif
```

Comment:
Changing the DB format depending if those are declared or not sounds a bad idea, as if one recompiles the code with different options, the DB format will change.